testing/ansible-lint/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Fabian Affolter <fabian@affolter-engineering.ch>
+# Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
+pkgname=ansible-lint
+pkgver=3.4.23
+pkgrel=0
+pkgdesc="A tool to check ansible playbooks"
+url="https://github.com/ansible/ansible-lint"
+arch="noarch"
+license="MIT"
+depends="python3 ansible py3-setuptools py3-six"
```

Comment:
I bet this should be just `depends="ansible py3-six"`
and next line `makedepends="python3-dev py3-setuptools"`

Reply:
btw There's issue about to split https://bugs.alpinelinux.org/issues/9622

Reply:
OK understand, see the benefit in moving setuptools to makedepends and get that python3 was redundant if ansible is listed just like py3-yaml was in fabaff's review. Thanks for your feedback!

I'm going to watch that bug about the split